pkg/scheduler/framework/plugins/noderesources/fit_test.go

```diff
@@ -736,3 +736,37 @@ func TestFitScore(t *testing.T) {
 		})
 	}
 }
+
+func TestEventsToRegister(t *testing.T) {
+	verifyEvent := func(el []framework.ClusterEvent, inPlaceVerticalScalingFeature bool) {
```

Comment:
Can we refactor this test a bit:

- change its style to be table-driven unit test
- create a []framework.ClusterEvent as field `want`, and use `cmp.Diff(want, got`) to compare the output, and thus simplify the logic.

Reply:
fixed in latest commit.